semanticdb/metacp/src/main/scala/scala/meta/internal/metacp/ToplevelClassfile.scala

```diff
@@ -4,6 +4,10 @@ import scala.meta.internal.io._
 import scala.meta.io._
 import scala.tools.asm.tree._
 
-final case class ToplevelClassfile(base: AbsolutePath, path: AbsolutePath, node: ClassNode) {
+final case class ToplevelClassfile(
+    base: AbsolutePath,
+    path: AbsolutePath,
+    node: ClassNode
+) {
```

Comment:
I'm surprised that Scalafmt doesn't undo these changes to formatting, now that `ToplevelClassfile` has three parameters again.

Reply:
Scalafmt keeps "config style" if it's there https://scalameta.org/scalafmt/#Configstyle It's nice to be able to force breaks even when the class could fit in a single line.

I manually undid these changes.